test/integration/adapters/mailgun_test.exs

```diff
@@ -0,0 +1,26 @@
+defmodule Swoosh.Integration.MailgunAdapterTest do
+  use ExUnit.Case, async: true
+
+  alias Swoosh.Adapters.Mailgun
+  import Swoosh.Email
+
+  @tag external: true
```

Comment:
Meant `@moduletag`

`@tag` is definitely for individual tests => http://elixir-lang.org/docs/stable/ex_unit/ExUnit.Case.html